File: DSA/DevvratKhandal_590014906/Day_2/Question1.java
public class Question1 {
    public static void main(String[] args) {
        int[] arr = {12, 35, 1, 10, 34, 1};

        int sum = 0;

        for (int i = 0; i < arr.length; i++) {
            sum += arr[i];
        }

        System.out.println("Sum of all elements in the array: " + sum);
    }
}

File: DSA/GaneshNikhil_590014521/question_1.cpp

// problem that introduces array traversal techniques and teaches how to efficiently process every
// element while maintaining a running total of values present in an array.
// Given an array of integers, the task is to find the sum of all elements in the array.
// This problem appears frequently in interviews and real-world applications like analytics process-ing,
// financial calculations, reporting systems, and data aggregation tasks. The challenge involves
// traversing the array efficiently and accumulating the result.
// This teaches:
// • array traversal
// • accumulation logic
// • variable tracking
// • optimization techniques
// which are essential for efficient data processing and interview problem solving.

#include<iostream>
#include<vector>

using namespace std;

int main(){
    int arr[20];
    int n;
    cout<<"Enter size: ";
    cin>>n;

    for(int i = 0 ; i < n ; i++){
        cin>>arr[i];
    }
    
    int total = 0;
    for(int i = 0 ; i < n ; i++){
        total += arr[i];
    }
    cout<<total<<endl;
}
